src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String REGEX = ",";
+    private static final String IS_SUPPLY = "supply";
+    private static final String IS_BUY = "buy";
```

Comment:
"is" before additional words means boolean type variable 
```suggestion
    private static final String SUPPLY = "supply";
    private static final String BUY = "buy";
```